R/paste_linter.R

```diff
@@ -1,13 +1,20 @@
 #' Raise lints for several common poor usages of `paste()`
 #'
 #' The following issues are linted by default by this linter
-#'   (and each can be turned off optionally):
+#'   (see options for which can be de-activated optionally):
```

Comment:
Options here sounds like something related to `options()`, or is it just me?

```suggestion
#'   (see arguments for which can be de-activated optionally):
```